packages/vuetifyjs.com/src/snippets/ts_breakpoint_example1.txt

```diff
@@ -20,4 +20,6 @@
   name: ('xs' | 'sm' | 'md' | 'lg' | 'xl')
   width: number
   height: number
+  thresholds: { xs: number, sm: number, md: number, lg: number }
```

Comment:
forgot `xl`?

Reply:
I remember leaving `xl` out, since it's implicitly defined by specifying the `lg` breakpoint: `xl` is just everything wider than that.